fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java

```diff
@@ -240,21 +319,34 @@ public static void main(String[] args) {
                 throw new Exception("Failed to get CosClient!");
             }
 
+            long num4Seg;
             S3Object item = cosClient.getObject(new GetObjectRequest(cosBucketName, cosFile2Break));
-            try (S3ObjectInputStream s3InStream = item.getObjectContent();
-                    BufferedReader resReader = new BufferedReader(new InputStreamReader(s3InStream))) {
-                   totalNum = m.getFhirResourceNumberFromBufferReader(resReader);
-               } catch (Exception ioe) {
-                   throw ioe;
-               }
-
             ByteArrayOutputStream bufferStream = new ByteArrayOutputStream();
```

Comment:
should still wrap this in a try-catch-close

Reply:
makes sense even though closing a ByteArrayOutputStream does nothing.

Reply:
done